Key.js

```diff
@@ -0,0 +1,81 @@
+const a = [];
```

Comment:
what the purpose of this file and `sass/abstract/_constants.scss`? As I see, they are useless